plug.vim

```diff
@@ -1053,14 +1053,16 @@ function! s:update_finish()
       if !pos
         continue
       endif
+      let shellerror = 0
       if has_key(spec, 'commit')
         call s:log4(name, 'Checking out '.spec.commit)
         let out = s:checkout(spec)
```

Comment:
`s:checkout` can fail. It used to set `v:shell_error` which is tested at line 1079, but now we have to update `shellerror`. The same goes for line 1072 and 1076.